lib/alex_koin/slack.ex

```diff
@@ -3,19 +3,29 @@ defmodule AlexKoin.SlackRtm do
     :alex_koin, :slack_module, Slack.Sends
   )
 
+  @koin_bot_id Application.get_env(:alex_koin, :koin_bot_id)
+
   alias AlexKoin.SlackCommands
 
   def handle_connect(slack, state) do
     IO.puts "Connected as #{slack.me.name}"
     {:ok, state}
   end
 
-  def handle_event(message = %{type: "message", text: "<@UC37P4L3Y> " <> text, user: user}, slack, state) do
-    SlackCommands.get_or_create(user)
-    |> create_reply(message, message_type(text)) # returns tuple {text, message_ts}
-    |> send_raw_message(message.channel, slack)
+  def handle_event(message = %{type: "message", channel: "D" <> _rest, user: user}, slack, state) do
```

Comment:
instead of `if ... else` here, you can just use a pattern match in the method signature: 

```elixir
def handle_event(%{user: @koin_bot_id, channel: "D" <> _rest}, _, state), do: {:ok, state}
def handle_event(message = %{type: "message", channel: "D" <> _rest, user: user}, slack, state) do
   handle_msg(user, message, message_type(message.text), slack, state)
end
```
This does two things: first, pattern matches are a bit faster than if statements, and also it isolates the two cases (passing only what the bot method cares about to it, and destructuring the rest where it's needed).